implementations/rust/ockam/ockam_command/src/enroll/command.rs

```diff
@@ -49,6 +47,10 @@ pub struct EnrollCommand {
     #[arg(long)]
     pub authorization_code_flow: bool,
 
+    /// Skip the enrollment process if the Identity is already enrolled
+    #[arg(long)]
```

Comment:
It would good to explain in the long help, when and why this argument should be used. 
Also the name is a bit confusing since it forces you to think in terms of double negation. I would understand better something like `force-enrollment` or simply `force`.

Reply:
Actually, the "correct" name for the argument would be "skip_oidc_user_enrollment_process". Also the "user_account_only" should be renamed to "skip_orchestrator_resources_creation" to be more precise.

Now, I'm not sure how to name them from the user perspective. Perhaps `--skip-user-enrollment` and `--skip-resources-creation`? The "force" behaviour is the default right now, which just goes through all the proceses.

Reply:
What if `force = false` was the default then? It seems to me that, by default, we don't want enrollment to happen if it's unnecessary, unless you truly want to re-enroll, possibly to use a different account (is there another use-case?). Then `force` makes sense to me.

I think that the second flag can be called `--skip-resource-creation`. I forgot that [I was the one proposing this feature ](https://github.com/build-trust/ockam/issues/5452) :-). This flag should probably stay hidden because, outside of our own testing, I'm not sure who would want to use it.

As usual naming is a bit subjective, so that's only my take on it :-).

Reply:
I've added this change in a separate commit. I think the UX is better this way, as we avoid prompting the user to do an action outside of the terminal when it's not necessary.